src/screens/User/User.tsx

```diff
@@ -45,21 +52,21 @@ const UserPage: React.FC<Props> = () => {
           <Image
             width={118}
             height={118}
-            source={{ uri: userPageInfo.user.image?.url }}
+            source={{ uri: user?.image?.url }}
```

Comment:
Do it for all, please.
```suggestion
            source={{ uri: user.image?.url }}
```